File: 1.HelloWorld/20160728ObjCWithSwift/ObjCWithSwift/ObjCWithSwift/ViewController.m
//
//  ViewController.m
//  ObjCWithSwift
//
//  Created by Financialbrain on 2016/7/28.
//  Copyright © 2016年 DarisCode. All rights reserved.
//

#import "ViewController.h"

///If you import multi Swift , only need to import below one

#import "ObjCWithSwift-Swift.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (void) viewDidAppear:(BOOL)animated
{
    
    [super viewDidAppear:animated];
    
    MySwiftClass * swiftObj = [MySwiftClass new];
    
    NSString * result = [swiftObj sayHello];
    NSDate * date = swiftObj.date;
    
    ///General Alert
    UIAlertController * alert = [UIAlertController alertControllerWithTitle:date.description message:result preferredStyle:UIAlertControllerStyleAlert];
    
    UIAlertAction * ok = [UIAlertAction actionWithTitle:@"OK" style:UIAlertActionStyleDefault handler:nil];
    
    [alert addAction:ok];
    [self presentViewController:alert animated:true completion:nil];
    
}

@end

File: HelloWorld/HelloWorld/ViewController.swift
//
//  ViewController.swift
//  HelloWorld
//
//  Created by Financialbrain on 2016/6/23.
//  Copyright © 2016年 DarisCode. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func showSwiftWorld()
{
        //Swift Main title / message
        let showAlert = UIAlertController(
            
            title: "Swift World",
            message: "Welcome to Swift World",
            preferredStyle: UIAlertControllerStyle.Alert
        )
        
        //Below Alart Message
        showAlert.addAction(
            
            UIAlertAction(
                title: "Finish",
                style: UIAlertActionStyle.Default,
                handler: nil
            )
        )
        //Show
        self.presentViewController(
            showAlert,
            animated: true,
            completion: nil
        )
    }
}


